rll/collectd.sh

```diff
@@ -259,11 +263,22 @@ else
   echo "swapfile not setup, skipping collectd swap plugin"
 fi
 
+# Populate RS_RLL_PORT
+source /var/run/rightlink/secret
+/usr/local/bin/rsc --rl10 rl10 put_hostname /rll/tss/hostname hostname=$COLLECTD_SERVER
```

Comment:
When using an rl10 call through RSC I believe it knows to use the RL10 creds already.  So you shouldn't need to source the secret file and you shouldn't need to pass the --rl10 flag.  The --rl10 flag is used to make calls to the cm15 (or other) APIs using the rl10 creds.